src/Polly.Extensions/Telemetry/EnrichmentContext.cs

```diff
@@ -5,6 +5,10 @@ namespace Polly.Extensions.Telemetry;
 /// </summary>
 public sealed partial class EnrichmentContext
 {
+    private const int InitialArraySize = 20;
```

Comment:
Just curious as to whether there's any meaning to this value, or if it's just arbitrary.

Reply:
Just arbitrary value I picked, for our use-cases it should be 6 at minimum